agent/config/config.go

```diff
@@ -438,6 +442,9 @@ func (cfg *Config) complete() bool {
 
 func fileConfig() (Config, error) {
 	fileName := utils.DefaultIfBlank(os.Getenv("ECS_AGENT_CONFIG_FILE_PATH"), defaultConfigFileName)
+	if runtime.GOOS == "windows" {
```

Comment:
Instead of checking the runtime OS, let's segregate the `defaultConfigFileName` constant in Windows and Linux/Unsupported files. This would ensure that we are not adding platform segregation logic in the common code.

Reply:
done, thanks